gremlin-go/driver/graphBinary.go

```diff
@@ -995,9 +1085,13 @@ func (serializer *graphBinaryTypeSerializer) getSerializerToRead(typ byte) (*gra
 		return &graphBinaryTypeSerializer{dataType: DurationType, reader: durationReader, nullFlagReturn: time.Duration(0), logHandler: serializer.logHandler}, nil
 	case MapType.getCodeByte():
 		return &graphBinaryTypeSerializer{dataType: MapType, writer: mapWriter, reader: mapReader, nullFlagReturn: nil, logHandler: serializer.logHandler}, nil
+	case BulkSetType.getCodeByte():
+		return &graphBinaryTypeSerializer{dataType: BulkSetType, reader: bulkSetReader, nullFlagReturn: nil, logHandler: serializer.logHandler}, nil
+	case TType.getCodeByte():
+		return &graphBinaryTypeSerializer{dataType: BulkSetType, reader: enumReader, nullFlagReturn: nil, logHandler: serializer.logHandler}, nil
 	default:
 		serializer.logHandler.logf(Error, deserializeDataTypeError, int32(typ))
-		return nil, errors.New("unknown data type to deserialize")
+		return nil, errors.New(fmt.Sprintf("unknown data type to deserialize %x", typ))
```

Comment:
can we just use fmt.Error here?
```suggestion
		return nil, fmt.Errorf("unknown data type to deserialize %x", typ)
```

Reply:
Ack. Changing in IDE